index.html

```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
+              <ul class="portfolio__list">
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p class="primary_description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
+        <article class="work_card">
+          <header class="work__header">
+            <img
+              src="assets/snapshoot2.png"
+              alt="Snapshoot Portfolio 2"
+              class="work__image"
+            />
+          </header>
+          <div class="portfolio__content">
+            <h3 class="portfolio__title color-dark">Multi-Post Stories</h3>
+            <div class="portfolio__profile">
+              <h4 class="portfolio__author color-info">Canopy</h4>
+              <ul class="portfolio__list">
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
+        <article class="work_card">
+          <header class="work__header">
+            <img
+              src="assets/snapshoot3.png"
+              alt="Snapshoot Portfolio 3"
+              class="work__image"
+            />
+          </header>
+          <div class="portfolio__content">
+            <h3 class="portfolio__title color-dark">Tonic</h3>
+            <div class="portfolio__profile">
+              <h4 class="portfolio__author color-info">Canopy</h4>
+              <ul class="portfolio__list">
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio__description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
```

Comment:
- [ ] "Nice work on the code! It appears to be a button with the text 'See project' inside it. Here are a few suggestions for improvement:

- [ ] Consider adding an href attribute to the <a> tag within the button. The href attribute specifies the URL that the button should link to. For example, <a href="project.html">See project</a>.